docs/extended/extensionsValuesOverwrite.md

```diff
@@ -0,0 +1,40 @@
+# Configuration Options for 'landscape.extensionsValuesOverwrite'
```

Comment:
If you agree to instead move this configuration to `landscape.gardener.extensions.valueOverwrites`, the docs/extended/gardener.md file would be the correct place for the documentation.

Reply:
I moved it to the gardener.md file